truss/parsepkgname/parsepkgname.go

```diff
@@ -0,0 +1,111 @@
+/*
+Package parsepkgname provides functions for extracting the name of a package
+from a protocol buffer definition file. For example, given a protocol buffer 3
+file like this:
+
+	// A comment about this proto file
+	package  examplepackage;
+
+	// and the rest of the file goes here
+
+The functions in this package would extract the name "examplepackage" as the
+name of the protobuf package.
+*/
+package parsepkgname
+
+import (
+	"io"
+	"unicode"
+
+	"github.com/TuneLab/go-truss/deftree/svcparse"
+)
+
+type Token int
+
+const (
+	IDENT Token = iota
```

Comment:
Is it important that these constants are exported? Idiomatic Go shies away from `ALL_CAPS_IDENTIFIERS`. I would have named them something like `tokenIdent`, `tokenWhitespace`, `tokenComment`, `tokenOther`. Prefixes are pretty Go-y (witness all the `ErrSomeError` errors stuff).

If the tokens *should* be exported, then never mind this; I'm not going to freak out about all-caps identifiers. Not too much, anyway.